webtau-groovy/src/test/groovy/com/twosigma/webtau/cfg/CLIArgsTest.groovy

```diff
@@ -0,0 +1,53 @@
+package com.twosigma.webtau.cfg
+
+import com.twosigma.webtau.utils.FileUtils
+import groovy.json.JsonOutput
+import org.junit.Test
+
+import java.nio.file.Paths
+import java.util.stream.Collectors
+
+class CLIArgsTest {
+    @Test
+    void "generate CLI docs"() {
+        try {
+            def handler = new WebTauGroovyCliArgsConfigHandler('--workingDir=${workingDir}', 'test.groovy')
+
+            WebTauConfig.registerConfigHandlerAsFirstHandler(handler)
+
+            def cfg = new WebTauConfig()
+
+            def cfgValues = cfg.getCfgValuesStream()
+            def cfgList = cfgValues.map { cfgValue ->
+                def defaultValue = cfgValue.key == 'workingDir' ? "" : (cfgValue.getAsString() ?: "")
+                return [
+                        name                  : cfgValue.key,
+                        'environment variable': cfgValue.prefixedUpperCaseKey,
+                        description           : cfgValue.description,
+                        'default value'       : defaultValue,
+                ]
+            }
+            .sorted { a, b ->
+                return a.name <=> b.name
+            }
+            .collect(Collectors.toList())
```

Comment:
FYI you could use groovy style operations

Reply:
WebTauConfig only lets me access the config values in a stream so may as well use stream operations rather than turning into a list and then doing groovy operations.

Reply:
makes sense. somehow I thought groovy way is available on streams.

Reply:
Good question, not sure, it might be though the mixture of meaning of collect would get a bit confusing :)